src/components/sbb-link/sbb-link.tsx

```diff
@@ -65,6 +65,12 @@ export class SbbLink {
    */
   @Prop() public textSize: InterfaceLinkAttributes['textSize'] = 's';
 
+  /**
+   * Set this property to true if you want only a visual represenation of a
+   * link, but no interaction (a span instead of a link will be rendered).
+   */
+  @Prop() public visualLinkOnly? = false;
```

Comment:
Yes this sounds better but there is the visualButtonOnly prop for the button and now a visualLinkOnly for the link :) You still want to change it?

Reply:
@kyubisation what do you think? Do we want to change in button too, or leave it? For me "link" or "button" should not be a part of the name.

Reply:
I just had a discussion with @HendrikExtSBB about this topic on `sbb-link` where we agreed on `is-static/isStatic`.
This might also be superseded by #1304